src/client/app/store/entity-metadata.ts

```diff
@@ -23,16 +23,18 @@ export function nameAndSayingFilter<T>(entities: T[], pattern: string) {
 
 export const entityMetadata: EntityMetadataMap = {
   Hero: {
+    entityName: 'Hero', // required
```

Comment:
we should comment (til we document) why this is required. Minification purposes

Reply:
We always needed it. It's just that I could derive it from the class ctor name before.

Reply:
Yes. But you changed it to a strong and it has consequences if it’s not a string. That’s where a comment or docs pointing to this example will help.